test/mri/tests/cext-ruby/symbol/noninterned_name.rb

```diff
@@ -0,0 +1,15 @@
+require "c/symbol"
+
+module Test_Symbol
+  module NonInterned
+    module_function
+
+    def noninterned_name(prefix = "")
+      prefix += "_#{Thread.current.object_id.to_s(36).tr('-', '_')}"
+      begin
+        name = "#{prefix}_#{rand(0x1000).to_s(16)}_#{Time.now.usec}"
+      end while Bug::Symbol.find(name)
+      name
+    end
+  end
+end
```

Comment:
How did this one get left behind?

Reply:
No idea, I just noticed it wasn't added after redoing the 2.3.7 import.